interfaces/builtin/avahi_observe.go

```diff
@@ -38,14 +236,19 @@ dbus (send)
     path=/
     interface=org.freedesktop.DBus.Peer
     member=Ping
-    peer=(name=org.freedesktop.Avahi,label=unconfined),
+    peer=(name=org.freedesktop.Avahi,label=###SLOT_SECURITY_TAGS###),
 
 dbus (send)
     bus=system
     path=/
     interface=org.freedesktop.Avahi.Server
-    member=Get*
-    peer=(name=org.freedesktop.Avahi,label=unconfined),
+    member={Get*,Resolve*,IsNSSSupportAvailable}
```

Comment:
when I was testing it with third party apps,  some are using instead of {Host/Address/Domain}Resolver direct resolve functions (ResolveHostName, ResolveAddress, ResolveService, ….). Then I was split to either use separate rules and add it under each respective resolving section, or to add it to existing Get* rule all at once. Similar for IsNSSSupportAvailable  which did not fit anywhere.  I did not have strong opinion here, smaller amounts of rules won eventually.
But happy to split them and add each to respective sections, if that improves readability.